tests/test_core.py

```diff
@@ -695,6 +695,8 @@ def test_stack_placement(self):
     self.do_run_in_out_file_test('tests', 'core', 'test_stack_placement')
 
   def test_stack_placement_pic(self):
+    if not self.is_wasm_backend() and self.get_setting('ALLOW_MEMORY_GROWTH'):
+      self.skipTest('memory growth is not compatible with MAIN_MODULE')
```

Comment:
But this doesn't doesn't use memory growth.   I guess maybe I was implicitly using it?

Reply:
Oh. maybe there is a memory growth test config that sets in globally.  Ignore this..